### PR TITLE
[Reviewer: Ellie] Remove unnecessary Juju/EC2 workarounds from Ellis charm

### PR DESCRIPTION
**Pull request by rkday-pro**
Our Juju charms have some code to work around the fact that, previously, `unit-get public-address` returned a split-horizon hostname rather than an IP address. In newer Juju versions, it now returns an IP address, so we can remove this code.

This removes it just from Ellis - once you've reviewed this, the changes to the other charms should be identical, so should need no review.

### CODE REVIEW

**Comment by rkday-pro** [comment]
Tested by deploying a bundle, then checking I could create a subscriber in Ellis.

File: charms/precise/clearwater-ellis/README.md [hunk 1]
```diff
@@ -63,8 +63,6 @@ Ellis does not currently support scaling out.
 
 The only currently supported DNS service for Clearwater is clearwater-route53 (when deploying in Amazon EC2), so will currently only work on EC2.  Additional DNS services will be added for other environments in future.
 
-Note that this charm does not current pass charm proof because its hooks use EC2 APIs directly to obtain the public IP address rather than unit-get public-address.  This is to work around what looks like a Juju bug - unit-get public-address on EC2 returns the public hostname of the unit rather than the public IP address.  Unfortunately EC2 public hostname are actually split horizon hostnames, so if you attempt to resolve the hostname from within EC2 it actually resolved to a private IP address - it will only resolve to a public address if resolved from outside EC2.
```
Comment (eleanor-merry) [inline charms/precise/clearwater-ellis/README.md:66]: Is the sentence above this out of date as well?

Comment (rkday-pro) [inline charms/precise/clearwater-ellis/README.md:66]: Yes, but it's not outdated by this pull request specifically - I'll fix all these READMEs up separately.

**Comment by eleanor-merry** [comment]
Looks good

**Pull request merged**